Packs/OpenLDAP/Integrations/OpenLDAP/OpenLDAP.py

```diff
@@ -108,12 +131,12 @@ def _get_tls_object(self):
             # Trust any certificate = False means that the LDAP server's certificate must be valid -
             # i.e if the server's certificate is not valid the connection will fail.
             tls = Tls(validate=ssl.CERT_REQUIRED, ca_certs_file=os.environ.get('SSL_CERT_FILE'),
-                      version=ssl.PROTOCOL_TLS)
+                      version=self._get_ssl_version())
```

Comment:
we dont want  ssl.PROTOCOL_TLS to be the default? We're passing None as default here

Reply:
No, I consulted with @glicht and we decided that None will be the default. 
Although I think that eventually None = PROTOCOL_TLS = PROTOCOL_SSLv23 (at least that is what I understood from the ssl code).